src/cider/type/plan/ConstantExpr.cpp

```diff
@@ -29,7 +29,7 @@ JITExprValue& Constant::codegen(CodegenContext& context) {
     return expr_var;
   }
 
-  auto null = func.createLiteral(JITTypeTag::BOOL, false);
+  auto null = func.createLiteral(JITTypeTag::BOOL, get_is_null());
```

Comment:
Is this for generate null constant?

Reply:
yes. In most cases constant is not null but except for `kNull`.